### PR TITLE
Allow specifying conf through env vars

### PR DESCRIPTION
**Pull request by vdbg**
Also allow for smaller conf files (only specify what needs to be overwritten)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,14 +11,17 @@ Allows for importing data from [Life360](https://www.life360.com/) to [InfluxDB]
 
 ## Setup
 
-### With Docker
+The app can be configured through config.toml, environment variables, or both.
```
Comment (Nickolaim) [inline README.md:14]: If a param is specified in both - who wins?

Comment (Nickolaim) [reply README.md:14]: Or add a note that the order is in the template file.

File: main.py [hunk 2]
```diff
@@ -9,49 +9,37 @@
 from influx import InfluxConnector
 from life360 import Life360Connector
 from pathlib import Path
+from config import Config
```
Comment (Nickolaim) [inline main.py:12]: Is it a new file?  Should it be added to the PR?

Comment (vdbg) [reply main.py:12]: oops. Added

File: README.md (hunk 1 above)
Comment (vdbg) [reply README.md]: done

**Review by Nickolaim — APPROVED** [review]

**Pull request merged**